services/udp/service.go

```diff
@@ -106,7 +106,7 @@ func (s *Service) writePoints() {
 		case batch := <-s.batcher.Out():
 			if err := s.PointsWriter.WritePoints(&cluster.WritePointsRequest{
 				Database:         s.config.Database,
-				RetentionPolicy:  "",
+				RetentionPolicy:  s.config.RetentionPolicy,
```

Comment:
Do you have to specify it in the config?  If you don't, will it `default` to the default one created?